src/components/static-map.js

```diff
@@ -174,16 +178,25 @@ export default class StaticMap extends PureComponent {
     this._mapboxMap = ref;
   }
 
+  // Handle map error
+  _mapboxMapError(evt) {
+    if (evt.error && evt.error.status === 401 && !this.state.accessTokenInvalid) {
```

Comment:
pull `401` out into a constant for better readability if it only signifies one thing? i.e.
```js
const UNAUTHORIZED_ERROR_CODE = 401;
```